developer-process/governance.md

```diff
@@ -0,0 +1,125 @@
+# vis.gl Community Governance Guidelines
```

Comment:
I wonder if a doc called "governance" should be at the top level or in a "governance" directory rather than inside "developer-process". 

Open Governance is what moving to UCF is all about... This doc represents the "big picture" that impact how users and contributors view visgl, where as a lot of the dev docs simply talks about details and practicalities around development.